YAFC/Windows/WelcomeScreen.cs

```diff
@@ -279,6 +279,11 @@ private enum EditType {
             gui.spacing = 1.5f;
         }
 
+        /// <summary>
+        /// <para>This initializes the different input fields with the supplied project definition. If the project is null, the fields are cleared.</para>
```

Comment:
When the method names are a verb in its base form -- `SetProject` -- the documentation of the method starts with a verb in its third form -- `Initializes the different [...]`.